seml/queue_experiments.py

```diff
@@ -170,7 +171,7 @@ def generate_configs(experiment_config):
     return all_configs
 
 
-def filter_experiments(collection, configurations):
+def filter_experiments(collection, configurations, no_hash=False):
```

Comment:
I would change that parameter to `use_hash`. Positives are less confusing. Of course, it has to be `no_hash` in `queue_experiments`, but we can switch it in this call.

Reply:
Thanks. Fixed in f3adc6d3266bc8a590dffcd77d448ce90bb2eb76